core/bind/core_bind.cpp

```diff
@@ -657,7 +657,7 @@ uint64_t _OS::get_unix_time_from_datetime(Dictionary datetime) const {
 	ERR_FAIL_COND_V(hour > 23, 0);
 
 	ERR_EXPLAIN("Invalid month value of: " + itos(month + 1));
-	ERR_FAIL_COND_V(month + 1 > 12, 0);
+	ERR_FAIL_COND_V(month > 11, 0); // 11, since it is 0-based
```

Comment:
Well, `month + 1 > 12` and `month > 11` are the same, no?

Reply:
Ok, read your explanations on #18221.

I'm fine with that change and it probably works around the bug, but it does not fix it. This will raise an error message like:
```
ERROR: Invalid month value of: 0
ERROR: ' 4294967295 > 11 ' is true
```

We should prevent `month` from wrapping around in the first place.

Reply:
Nope, it shows only the first ERROR: line, not the second:
![image](https://user-images.githubusercontent.com/5276727/38858746-2a2d872e-4235-11e8-87e6-c465a97a7d7b.png)
![image](https://user-images.githubusercontent.com/5276727/38858787-43d08cbc-4235-11e8-81e0-1a4e50b3e402.png)
If ERR_FAIL_INDEX_V was used instead, it would have shown the huge unsigned value.

Reply:
Well that error log is still misleading:
```
Description: Invalid month value of: 0
C Error: Condition ' month > 11 ' is true.
```
EMAKESNOSENSE :P